docs/content/en/docs/Getting started/setup-development/index.md

```diff
@@ -8,7 +8,7 @@ description: >
 
 There are three options to setup a development environment:
 1. using a seperate device in combination with k3OS and our installation script (preferred). This requires an external device and is a fully automated installation.
-2. using [minikube](https://kubernetes.io/en/docs/setup/minikube/) (recommended for developers working on the core functionalities of the stack). This method allows you to install the stack on your device and is semi-automated.
+2. using [minikube](https://kubernetes.io/de/docs/setup/minikube/) (recommended for developers working on the core functionalities of the stack). This method allows you to install the stack on your device and is semi-automated.
```

Comment:
Do we really want to link the german documentation, inside an english "Getting started" guide ?

Reply:
Good catch!

Reply:
Opened a new issue for this: #561